shaders/motion_blur.shader

```diff
@@ -0,0 +1,3 @@
+//=========================================================================================================================
+// NOT USED YET!!
+//=========================================================================================================================
```

Comment:
'Empty' files should not be added to the project, please omit this file.

Reply:
Updated!